template/src/index.template.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
```

Comment:
We should always have a lang attribute here. Value should be determined by country site is for. I'd add "en-us" value as default.